release-notes/v0.5.0.md

```diff
@@ -0,0 +1,95 @@
+# What's new in VS Code Java?
+
+*November 2018*
+
+Welcome to the first issue of VS Code Java release notes. This document aims to recap the features, bug fixes around Java language support in the past month. Here are some of the key highlights:
+
+- **[Rename refactoring](#rename-refactoring)** supports file rename/move
+- **[IntelliCode](#intellicode)**, AI-assisted coding experience now supports Java
+- **[Logpoints](#logpoints)** supported by the debugger
```

Comment:
swap its order with the next one. 
AND add `Run or debug using codelens` in highlight bullets.

Reply:
Not all changes need to be highlighted. This is more of an enhancement.